test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class DefaultIncludeTest < ActiveSupport::TestCase
+      class Blog < ActiveModelSerializers::Model
+        attr_accessor :id, :name, :posts
+      end
+      class Post < ActiveModelSerializers::Model
+        attr_accessor :id, :title, :author
+      end
+      class Author < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+      class BlogSerializer < ActiveModel::Serializer
+        attributes :id
+        attribute :name, key: :title
+
+        has_many :posts
+      end
+      class PostSerializer < ActiveModel::Serializer
+        default_include [:author]
+        attributes :id, :title
+
+        has_one :author
```

Comment:
@NullVoxPopuli 

> but what does include all mean? include all dependencies? include all authors?
> if anything, include: :always would make more sense.

`:all`, meaning 'all fields', was really just an attempt at something better than `true` and something like for all attributes `include: true` or limit them with `include: true, fields: [:name]`.  I think there's definitely a better way to phrase this, but main point is location of the new api and the kinds of words to use.  

So that we could exchange a line in the controller

```ruby
    render json: post, include: [:metrics, :media, :publishes, :author],
        fields: { media: [:title], publishers: (PublisherSerializer._attributes - :publisher_bio) }
```

with

```ruby
class PostSerializer
  has_many :metrics, include: :all
  has_one :media, include: [:title]
  has_many :publishers, exclude: [:publisher_bio]
end
render json: post
```